Mathlib/Topology/Order/IsNormal.lean

```diff
@@ -0,0 +1,52 @@
+/-
+Copyright (c) 2025 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+import Mathlib.Order.IsNormal
+import Mathlib.Topology.Order.IsLUB
+
+/-!
+# A normal function is strictly monotone and continuous
+
+We defined the predicate `Order.IsNormal` in terms of `IsLUB`, which avoids having to import
+topology in order theory files. This file shows that the predicate is equivalent to the definition
+in the literature, being that of a strictly monotonic function, continuous in the order topology.
+-/
+
+open Set
+
+namespace Order
+variable {α β : Type*}
+  [LinearOrder α] [WellFoundedLT α] [TopologicalSpace α] [OrderTopology α]
+  [LinearOrder β] [WellFoundedLT β] [TopologicalSpace β] [OrderTopology β]
+
+attribute [local instance]
+  WellFoundedLT.toOrderBot WellFoundedLT.conditionallyCompleteLinearOrderBot in
+theorem IsNormal.continuous {f : α → β} (hf : IsNormal f) : Continuous f := by
+  rw [OrderTopology.continuous_iff]
+  refine fun b ↦ ⟨?_, ((isLowerSet_Iio b).preimage hf.strictMono.monotone).isOpen⟩
+  rw [← isClosed_compl_iff, ← Set.preimage_compl, Set.compl_Ioi]
+  obtain ha | ⟨a, ha⟩ := ((isLowerSet_Iic b).preimage hf.strictMono.monotone).eq_univ_or_Iio
+  · exact ha ▸ isClosed_univ
```

Comment:
```suggestion
  obtain ha | ⟨a, rfl⟩ := ((isLowerSet_Iic b).preimage hf.strictMono.monotone).eq_univ_or_Iio
  · exact isClosed_univ
```
and similar below, I guess

Reply:
I don't think this works, since `Iic b` isn't a single variable.